templates/new_home/contact.html

```diff
@@ -8,7 +8,7 @@ <h1 data-aos="fade-up">Contact Us</h1>
   <div class="row">
   
     <div class="col-md-6 px-0">
-      <div class="card">
+      <div class="shadow-lg p-3 mb-5 bg-white rounded card-rounder">
```

Comment:
`rounded` not required with `card-rounder` class.

Reply:
done !